Week01/Problem04/shashemi_04.py

```diff
@@ -0,0 +1,12 @@
+palindrome=[]
+for i in range(1000,100,-1):
+    for j in range(1000,i,-1):
+        candidate=i*j
+        tst=str(candidate)
+        if len(tst)%2==0:
+            if tst[-1]==tst[0] and tst[-2]==tst[1] and tst[-3]==tst[2]:
+                palindrome.append(candidate)
+        else:
+            if tst[-1]==tst[0] and tst[-2]==tst[1]:
+                palindrome.append(candidate)
```

Comment:
It's often best to try and write code that is a little more general than this. It's okay to do this when you're just starting out though.